manifests/v1beta1/components/controller/controller.yaml

```diff
@@ -51,18 +51,11 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
           volumeMounts:
-            - mountPath: /tmp/cert
-              name: cert
-              readOnly: true
             - mountPath: /katib-config.yaml
               name: katib-config
               subPath: katib-config.yaml
               readOnly: true
       volumes:
-        - name: cert
-          secret:
-            defaultMode: 420
-            secretName: katib-webhook-cert
```

Comment:
Oh, it's a good point. In this implementation, cert-manager integration doesn't work...
So we should deploy `katib-webhook-cert` without any certs, and then the cert-generator injects certs into the mounted cert. This means we shouldn't create a secret within the cert-generator.

Reply:
Done: https://github.com/kubeflow/katib/pull/2185/commits/ce64c4d9ee041278231de81b19bd105ea60c1530

Reply:
@tenzen-y I am wondering, do we really need to have this secret: `katib-webhook-cert` if certs will be injected (to controller volume and webhook `caBundle`) during controller initialization step ? 
I think, storing certs in secret will be useful to avoid cert regeneration when Katib Controller is restarted.
Is there anything else ?

Reply:
> I am wondering, do we really need to have this secret: katib-webhook-cert if certs will be injected (to controller volume and webhook caBundle) during controller initialization step ?

I think it is needed since we can avoid to re-generate certs as you say.

> I think, storing certs in secret will be useful to avoid cert regeneration when Katib Controller is restarted.
Is there anything else ?

As one more advantage, we can set up certs for the webhook using the same architecture in the case of using the cert-manager. (The cert-manager will inject certs to secret and webhookConfigurations.)

By using the same architecture across every cert creation, we can avoid inventing a new concept and using the same architecture is useful for debugging in the event of bugs regarding webhooks.

Reply:
Sounds good. Let's keep secret implementation then.